### ISSUE TITLE
The last point isn't visible when updating the marker symbol with the accessibility module attached

### ISSUE THREAD

**Issue by mfilipiec** [highcharts/highcharts#16235]
#### Expected behaviour
All the points are visible.

#### Actual behaviour
Like in the title (its opacity is set to 0).

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/e0ngtwy8/

Workaround: fire additional `chart.redraw()` https://jsfiddle.net/BlackLabel/gnko9wst/

#### Product version
v9.2.1

**Comment by raf18seb** [comment]
FYI @oysteinmoseng

**Comment by raf18seb** [comment]
Closing as a probably duplicate of https://github.com/highcharts/highcharts/issues/15480 and https://github.com/highcharts/highcharts/issues/16052
